examples/all-clusters-app/all-clusters-common/src/electrical-power-measurement-stub.cpp

```diff
@@ -47,5 +47,7 @@ void emberAfElectricalPowerMeasurementClusterInitCallback(chip::EndpointId endpo
                 OptionalAttributes::kOptionalAttributeNeutralCurrent));
 
         gEPMInstance->Init();
+
+        gEPMDelegate->SetPowerMode(PowerModeEnum::kAc);
```

Comment:
All of the attributes in all-clusters-app were already switched on (in zap), and all features were on.
The cluster here handles the FeatureMap (not in ZAP).

![image](https://github.com/project-chip/connectedhomeip/assets/93921463/13508093-2586-4ab9-8724-ea104ea4ccf1)

The test case EPM_2_1 failed because the PowerMode was defaulted to Unknown

Reply:
If you run IDM-10.2, does it pass? https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/TC_DeviceConformance.py

Reply:
Now that the updated DataModel XML has been merged, then the EEM and EPM clusters are clean